public/services/actions/webhook.ts

```diff
@@ -0,0 +1,43 @@
+import { http, Result, StringObject } from "@fider/services"
+import { HttpHeaders, WebhookPreviewResult, WebhookStatus, WebhookTriggerResult, WebhookType } from "@fider/models"
+
+export const createWebhook = async (
```

Comment:
because of the long list of params here, would it be worth having a type for it?

Reply:
I think it was the reason why I made an extra (maybe already existing) TypeScript type definition, but I kinda messed up with it to be honest. I'll try to improve this little issue